shm/Fruit.hpp

```diff
@@ -2,13 +2,17 @@
 
 class Fruit : public Cargo {
 public:
+    Fruit(const std::string& name, size_t amount, size_t basePrice)
+        : Cargo(name, amount, basePrice) {}
+
     //override from Cargo
     std::string getName() const override { return name_; }
     size_t getAmount() const override { return amount_; }
     size_t getBasePrice() const override { return basePrice_; }
     size_t getPrice() const override {
         return static_cast<size_t>(basePrice_ * daysToRot_ / expirationDays_);
     }
+    void nextDay() override { --(*this); }
```

Comment:
Could we do only `void nextDay () ovveride;` ?

Reply:
What do you mean? Without method's body?

Reply:
@lisie-sprawy we can have `void nextDay () ovveride;` here in hpp file, but then the method's body have to be in cpp file.